src/main/drivers/pwm_mapping.h

```diff
@@ -36,9 +36,15 @@
 #define MAX_INPUTS  8
 
 #define PWM_TIMER_MHZ 1
-#define ONESHOT125_TIMER_MHZ 4
 #define ONESHOT42_TIMER_MHZ 12
-#define MULTISHOT_TIMER_MHZ 12
+#define ONESHOT125_TIMER_MHZ 8
+#define ONESHOT42_TIMER_MHZ 12
```

Comment:
Sorry was not clear here before. Value need to be 24 (oneshot42 3 times faster then oneshot125)